vnext/Microsoft.ReactNative/ReactApplication.cpp

```diff
@@ -131,16 +132,19 @@ void ReactApplication::OnLaunched(LaunchActivatedEventArgs const &e) {
   this->OnCreate(e);
 }
 
+void ApplyArguments(ReactNative::ReactNativeHost const &host, std::wstring const &arguments) noexcept;
+
 /// <summary>
 /// Invoked when the application is launched normally by the end user.  Other
 /// entry points will be used such as when the application is launched to open a
 /// specific file.
 /// </summary>
 /// <param name="e">Details about the launch request and process.</param>
 void ReactApplication::OnCreate(IActivatedEventArgs const &e) {
-  if (!m_delegate) {
-    m_delegate = CreateReactApplicationDelegate();
-  }
+  // Resuming({this, &ReactApplicationDelegate::OnResuming});
+  // Suspending({this, &ReactApplicationDelegate::OnSuspending});
+  // LeavingBackground({this, &ReactApplicationDelegate::OnLeavingBackground});
+  // EnteredBackground({this, &ReactApplicationDelegate::OnEnteredBackground});
```

Comment:
It would be great if we can redirect all these events to the ReactNotificationService for which I am going to send a PR today.

Reply:
@vmoroz can you clarify what you'd suggest? the methods in the delegate weren't really doing anything

Reply:
In future we can raise these events from the ReactNotificationSevice. For now, I would propose to delete this code to avoid having commented out "dead" code.

---
In reply to: [429664370](https://github.com/microsoft/react-native-windows/pull/4949#discussion_r429664370) [](ancestors = 429664370)